ajax/libs/redux-persist/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "redux-persist",
+  "version": "4.0.0-alpha6",
+  "description": "persist and rehydrate redux stores",
+  "main": "lib/index.js",
```

Comment:
@ian4hu Please remove the fields we don't need like `"main"`, `"module"`, `"jsnext:main"`, `"ava"`, `"files"`, `"standard"`.